code/modules/mob/living/simple_animal/hostile/beheaded_kamikaze.dm

```diff
@@ -0,0 +1,30 @@
+/mob/living/simple_animal/hostile/beheaded_kamikaze
+	name = "beheaded kamikaze"
+	desc = "You can't be serious."
+	icon_state = "clown" //placeholder
+	icon_living = "clown" //placeholder
+	speak_chance = 80
+	speak = list("AAAAAAAAAAAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAAAA")
+	emote_taunt = list("screams", "yells")
+	taunt_chance = 80
+	speed = 0
+	maxHealth = 7
```

Comment:
Punching it takes around 3 hits. Toolboxes two. Anything else might or might not kill it in one hit.